python/dataproc_templates/databases/README.md

```diff
@@ -0,0 +1,85 @@
+# Dataproc Serverless: Databases To GCS
+
+Template for exporting a Database table to files in Google Cloud Storage. It supports writing JSON, CSV, Parquet and Avro formats for Postgresql, MySql, MS SQL Server (GCP and onprem).
+
+## Arguments
```

Comment:
please update template to support following configurations: partitionColumn, lowerBound, upperBound and numPartitions